### PR TITLE
[FEAT] Add new section for News

### PR DESCRIPTION
**Pull request by csouchet**

### CODE REVIEW

File: src/content/NewsContent.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * limitations under the License.
+ */
+import { PostDescription } from '../theme/types';
+
+const news: PostDescription[] = [
+  {
+    title: 'August 2021 Newsletter',
+    text: 'Learn about the new Process Analytics website and logo, the new R library and improvements in the BPMN Visualization Typescript library.',
+    cover: 'https://miro.medium.com/max/350/0*Yymlf2vjfAIINqc2',
+    url: 'https://medium.com/@process-analytics/process-analytics-august-2021-newsletter-b793a647e35d',
```
Comment (tbouffard) [inline src/content/NewsContent.tsx:23]: ⚠️ this image is currently too large: 191kb
for posts, we have images from 16kb to 113kb. 

See also #30 and #131

Comment (csouchet) [reply src/content/NewsContent.tsx:23]: I reduced to 300. I I reduce more, the size rendered is smaller 🙁

Comment (tbouffard) [reply src/content/NewsContent.tsx:23]: 137kb now. I guess we should manage the image resize directly on the medium article.

File: src/theme/sections/News.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import React from 'react';
+import { Fade } from 'react-awesome-reveal';
+import Section from '../components/Section';
+import { CardContainer } from '../components/Card';
+import Triangle from '../components/Triangle';
+import { Post } from '../components/Post';
+import { SECTION } from '../utils/constants';
+import { newsContent } from '../../content/NewsContent';
+
+const cardMinWidth = '350px';
```
Comment (tbouffard) [inline src/theme/sections/News.tsx:25]: ℹ️ we could probably shared the value with the `Blog` component

**Review by tbouffard — APPROVED** [review]
Nice job 🎉

**Pull request merged**